app/frontend/src/pages/CreditsPage/CreditsPage.tsx

```diff
@@ -0,0 +1,138 @@
+// External Imports
+import React, { Fragment, useState } from "react";
+
+// Internal Imports
+import {
+  logoHorizontal,
+  logoStacked,
+  logoHorizontalOnDark,
+  logoStackedOnDark,
+  logoHfLA,
+} from "assets/images/images";
+import { HeaderNav, FooterNav, Button } from "components/components";
+import { Card } from "components/Cards/StandardCard";
+import { iconData } from "pages/api_data/creditsIconData";
+import { illustrationData } from "pages/api_data/creditsIllustrationData";
+
+const CreditsPage = () => {
+  const [activeData, setActiveData] = useState(illustrationData);
+  const [activeButton, setActiveButton] = useState("btnIllustration");
+
+  const handleClickBtnIllustration = () => {
+    setActiveButton("btnIllustration");
+    setActiveData(illustrationData);
+  };
+
+  const handleClickBtnIcon = () => {
+    setActiveData(iconData);
+    setActiveButton("btnIcon");
+  };
+
+  return (
+    <>
+      <HeaderNav
+        logoDesktop={logoHorizontal}
+        logoMobile={logoStacked}
+        menu={[
+          { name: "Hack for LA", link: "/" },
+          { name: "How to Join", link: "/qualifier/1" },
+          { name: "Projects", link: "/demo" },
+        ]}
+      />
+      <main>
+        <section className="body-container">
+          <div className="main-container">
+            <div className="text-container">
+              <h1 className="credits-intro-title mt-0 mb-3">Credits</h1>
+              <p className="paragraph-1 credits-intro-paragraph">
+                Thank you to all of the artists and sponsors who help make our
+                projects successful. Check out all of the illustrations and
+                iconography we have used on our site.
+              </p>
+            </div>
+
+            <div className="pt-5 mt-5">
+              <h2 className="credits-test">Illustrations & Iconography</h2>
+              <div>
+                <Button
+                  color="primary"
+                  size="md"
+                  onClick={handleClickBtnIllustration}
+                  addClass={`mr-3 btn-border ${
+                    activeButton === "btnIllustration" ? "" : "btn-outline"
```

Comment:
Is there a reason for using Buttons rather than a Chip component? It looks like the component is a Chip in figma.

Reply:
No, the reason I used buttons was because I didn't know they were supposed to be chips. I'll make the changes. Where in Figma should I be checking to see what components a design has used? I seem to have completed missed this info.

Reply:
Gotcha. You can view the component (sometimes, not all the time) by clicking on it and viewing the right side panel.

<img width="800" alt="Screenshot 2023-05-10 at 01 57 01 PM" src="https://github.com/hackforla/CivicTechJobs/assets/62368440/9fb58233-d088-48a8-8700-b80c42d0915a">

Reply:
Hmm I still don't see that information (my apologies! I don't have a lot of experience using Figma): 

<img width="1491" alt="Screenshot 2023-05-10 at 3 30 00 PM" src="https://github.com/hackforla/CivicTechJobs/assets/86077274/24ea5428-d835-46f4-8882-325c09e66204">

Reply:
Looks like your box was centered in the text inside of the component. It should work if your hover the mouse closer to the border of the chip!

Reply:
I think I found the reason--I was looking at the final page design in "MVP DESIGNS" instead of "DEV HANDOFF". I'll keep this in mind for future work involving Figma.